### PR TITLE
:white_check_mark: add in all passing istanbul ethereum tests

### PR DESCRIPTION
**Pull request by zander-xyz**

### CODE REVIEW

File: scripts/ethereum-test-filter/filter [hunk 1]
```diff
@@ -3,4 +3,4 @@ homestead -VMTests/vmIOandFlowOperations.jumpToPush
...
...icideEnd:stCallCodes.callcallcallcode_001_OOGMBefore:stCallCodes.callcall_00_OOGE:stCallCodes.callcallcall_000_OOGMAfter:stCallCodes.callcallcodecallcode_011_SuicideMiddle:stCallCodes.callcall_00_OOGE_valueTransfer:stCallCodes.callcallcodecallcode_011_OOGE:stBugs.staticcall_createfails:stCallCodes.callcallcodecall_ABCB_RECURSIVE:stCallCodes.callcallcodecallcode_011_OOGMAfter:stCallCodes.callcallcodecallcode_011_OOGMBefore:stCallCodes.call_OOG_additionalGasCosts2:stCallCodes.callcallcall_ABCB_RECURSIVE:stBugs.evmBytecode:VMTests/vmTests.sha3:stArgsZeroOneBalance.suicideNonConst:VMTests/vmIOandFlowOperations.jump:stAttackTest.ContractCreationSpam:VMTests/vmLogTest.log3:stAttackTest.CrashingTransaction:VMTests/vmLogTest.log4:VMTests/vmLogTest.log1:VMTests/vmLogTest.log0:VMTests/vmLogTest.log2:VMTests/vmTests.calldatacopy:VMTests/vmTests.random:VMTests/vmTests.suicide:stArgsZeroOneBalance.jumpiNonConst:VMTests/vmArithmeticTest.mul:stArgsZeroOneBalance.jumpNonConst:stArgsZeroOneBalance.delegatecallNonConst:stArgsZeroOneBalance.callcodeNonConst:VMTests/vmIOandFlowOperations.return:stArgsZeroOneBalance.callNonConst:VMTests/vmIOandFlowOperations.codecopy:VMTests/vmIOandFlowOperations.mload
```
Comment (zander-xyz) [inline scripts/ethereum-test-filter/filter:6]: We'll remove tests from this filter as they get fixed

Comment (sheapwilliams) [reply scripts/ethereum-test-filter/filter:6]: Sweet Jesus - even GitHub gave up on reading this diff...we prolly need multi-line white list support?

Comment (zander-xyz) [reply scripts/ethereum-test-filter/filter:6]: Sublime makes this much easier to read. Find all ":", press enter gives something very pretty. But yes, I agree.

Comment (rgarc) [reply scripts/ethereum-test-filter/filter:6]: Perhaps the massive line serves as an incentive to fix tests ASAP? 🔥

**Pull request merged**